lib/resty/radixtree.lua

```diff
@@ -703,8 +703,23 @@ function _M.dispatch(self, path, opts, ...)
             self.args = {...}
         else
             clear_tab(self.args)
-            for i = 1, len do
-                self.args[i] = select(i, ...)
+            self.args[1] = select(1, ...)
```

Comment:
I do not know the reason: why is the new way better?

Reply:
ping @wangxiaochuang

Reply:
luajit NYI for select: Only compiled when first argument is a constant (Has to be positive if used with varg).
maybe I was wrong...